### DISCUSSION TITLE
Login to Jagex Account using locally built client?

### DISCUSSION THREAD

**Discussion by lucas-bastosf** [runelite/runelite#17974] [Q&A] [answered]
Hi. 

Is it possible to login into a Jagex Account, with a locally built (from source) client? Or maybe a way to link the Jagex Launcher to use this custom client?

Thanks..

**Comment by DapperMickie** [comment, marked as answer]
https://github.com/runelite/runelite/wiki/Using-Jagex-Accounts

**Comment by lucas-bastosf** [comment]
Thank you! That's exactly what I needed.
